src/proj.rs

```diff
@@ -107,24 +107,31 @@ pub enum ProjError {
     Network,
     #[error("Could not set remote grid download callbacks")]
     RemoteCallbacks,
-    #[error("Couldn't build request")]
+    #[error("Couldn't access the network")]
     #[cfg(feature = "network")]
-    BuilderError(#[from] reqwest::Error),
+    NetworkError(Box<ureq::Error>),
```

Comment:
Let's mark `ProjError` as `#[non_exhaustive]`, so we don't take a breaking change next time we add a variant.